bundle.mustache

```diff
@@ -18,6 +18,8 @@ BuildRequires: %{?scl_prefix_nodejs}npm
 %else
 BuildRequires: nodejs-packaging
 %endif
+BuildRequires: nodejs
+BuildRequires: npm
```

Comment:
I think we'd want to move these inside the else block above so they aren't installed when built with an SCL

Reply:
`if` has `%{?scl_prefix_nodejs}npm`. I guess that on EL7 `node-packaging` pulls in `npm`. Note point 3 in https://github.com/theforeman/npm2rpm/issues/17 btw.

Reply:
My suggestion would be
```
BuildRequires: %{?scl_prefix_nodejs}npm
%if 0%{!?scl:1}
BuildRequires: nodejs-packaging
%endif
```